base_import_pdf_by_template/views/base_import_pdf_template_line_views.xml

```diff
@@ -19,68 +19,69 @@
             <form>
                 <sheet>
                     <group name="main_group">
-                        <group name="left">
```

Comment:
Why removing groups? At first sight, I prefer to have it into 2 columns.

Reply:
I think an example may help to understand it

**Before**
![antes](https://github.com/user-attachments/assets/76fcdd35-7eab-4f72-9062-a19b95a6206c)

**After**
![despues](https://github.com/user-attachments/assets/a5cac3d2-a17e-4457-90e4-619283bb15d3)